### ISSUE TITLE
为什么安装好之后网页能够登录，但是一片空白。

### ISSUE THREAD

**Issue by Mikasal** [naurril/SUSTechPOINTS#132]
你好，作者，很感谢你开源了这个工具。为我的工作带来了很多便利。但是现在我在一台电脑上可以正常使用，但是在另外一台电脑安装好之后能打开网页，但是网页一篇空白，能解答一下吗？

**Comment by naurril** [comment]
需要在console看一下有没有什么报错

**Comment by Mikasal** [comment]
`2023-01-16 10:19:32.475409: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
Model: "model"
__________________________________________________________________________________________________
Layer (type)                    Output Shape         Param #     Connected to
==================================================================================================
input_1 (InputLayer)            [(None, 512, 3)]     0
__________________________________________________________________________________________________
tf_op_layer_ExpandDims (TensorF (None, 512, 3, 1)    0           input_1[0][0]
__________________________________________________________________________________________________
conv2d (Conv2D)                 (None, 512, 1, 64)   256         tf_op_layer_ExpandDims[0][0]
__________________________________________________________________________________________________
batch_normalization (BatchNorma (None, 512, 1, 64)   256         conv2d[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu (TensorFlowOpL (None, 512, 1, 64)   0           batch_normalization[0][0]
__________________________________________________________________________________________________
conv2d_1 (Conv2D)               (None, 512, 1, 128)  8320        tf_op_layer_Relu[0][0]
__________________________________________________________________________________________________
batch_normalization_1 (BatchNor (None, 512, 1, 128)  512         conv2d_1[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_1 (TensorFlowO (None, 512, 1, 128)  0           batch_normalization_1[0][0]
__________________________________________________________________________________________________
conv2d_2 (Conv2D)               (None, 512, 1, 1024) 132096      tf_op_layer_Relu_1[0][0]
__________________________________________________________________________________________________
batch_normalization_2 (BatchNor (None, 512, 1, 1024) 4096        conv2d_2[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_2 (TensorFlowO (None, 512, 1, 1024) 0           batch_normalization_2[0][0]
__________________________________________________________________________________________________
max_pooling2d (MaxPooling2D)    (None, 1, 1, 1024)   0           tf_op_layer_Relu_2[0][0]
__________________________________________________________________________________________________
tf_op_layer_Squeeze (TensorFlow (None, 1024)         0           max_pooling2d[0][0]
__________________________________________________________________________________________________
dense (Dense)                   (None, 512)          524800      tf_op_layer_Squeeze[0][0]
__________________________________________________________________________________________________
batch_normalization_3 (BatchNor (None, 512)          2048        dense[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_3 (TensorFlowO (None, 512)          0           batch_normalization_3[0][0]
__________________________________________________________________________________________________
dense_1 (Dense)                 (None, 256)          131328      tf_op_layer_Relu_3[0][0]
__________________________________________________________________________________________________
batch_normalization_4 (BatchNor (None, 256)          1024        dense_1[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_4 (TensorFlowO (None, 256)          0           batch_normalization_4[0][0]
__________________________________________________________________________________________________
dense_2 (Dense)                 (None, 9)            2313        tf_op_layer_Relu_4[0][0]
__________________________________________________________________________________________________
tf_op_layer_Add (TensorFlowOpLa (None, 9)            0           dense_2[0][0]
__________________________________________________________________________________________________
tf_op_layer_Reshape (TensorFlow (None, 3, 3)         0           tf_op_layer_Add[0][0]
__________________________________________________________________________________________________
tf_op_layer_MatMul (TensorFlowO (None, 512, 3)       0           input_1[0][0]
                                                                 tf_op_layer_Reshape[0][0]
__________________________________________________________________________________________________
tf_op_layer_ExpandDims_1 (Tenso (None, 512, 3, 1)    0           tf_op_layer_MatMul[0][0]
__________________________________________________________________________________________________
conv2d_3 (Conv2D)               (None, 512, 1, 64)   256         tf_op_layer_ExpandDims_1[0][0]
__________________________________________________________________________________________________
batch_normalization_5 (BatchNor (None, 512, 1, 64)   256         conv2d_3[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_5 (TensorFlowO (None, 512, 1, 64)   0           batch_normalization_5[0][0]
__________________________________________________________________________________________________
conv2d_4 (Conv2D)               (None, 512, 1, 64)   4160        tf_op_layer_Relu_5[0][0]
__________________________________________________________________________________________________
batch_normalization_6 (BatchNor (None, 512, 1, 64)   256         conv2d_4[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_6 (TensorFlowO (None, 512, 1, 64)   0           batch_normalization_6[0][0]
__________________________________________________________________________________________________
conv2d_5 (Conv2D)               (None, 512, 1, 64)   4160        tf_op_layer_Relu_6[0][0]
__________________________________________________________________________________________________
batch_normalization_7 (BatchNor (None, 512, 1, 64)   256         conv2d_5[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_7 (TensorFlowO (None, 512, 1, 64)   0           batch_normalization_7[0][0]
__________________________________________________________________________________________________
conv2d_6 (Conv2D)               (None, 512, 1, 128)  8320        tf_op_layer_Relu_7[0][0]
__________________________________________________________________________________________________
batch_normalization_8 (BatchNor (None, 512, 1, 128)  512         conv2d_6[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_8 (TensorFlowO (None, 512, 1, 128)  0           batch_normalization_8[0][0]
__________________________________________________________________________________________________
conv2d_7 (Conv2D)               (None, 512, 1, 1024) 132096      tf_op_layer_Relu_8[0][0]
__________________________________________________________________________________________________
batch_normalization_9 (BatchNor (None, 512, 1, 1024) 4096        conv2d_7[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_9 (TensorFlowO (None, 512, 1, 1024) 0           batch_normalization_9[0][0]
__________________________________________________________________________________________________
max_pooling2d_1 (MaxPooling2D)  (None, 1, 1, 1024)   0           tf_op_layer_Relu_9[0][0]
__________________________________________________________________________________________________
tf_op_layer_Squeeze_1 (TensorFl (None, 1024)         0           max_pooling2d_1[0][0]
__________________________________________________________________________________________________
dense_3 (Dense)                 (None, 512)          524800      tf_op_layer_Squeeze_1[0][0]
__________________________________________________________________________________________________
batch_normalization_10 (BatchNo (None, 512)          2048        dense_3[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_10 (TensorFlow (None, 512)          0           batch_normalization_10[0][0]
__________________________________________________________________________________________________
dense_4 (Dense)                 (None, 256)          131328      tf_op_layer_Relu_10[0][0]
__________________________________________________________________________________________________
batch_normalization_11 (BatchNo (None, 256)          1024        dense_4[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_11 (TensorFlow (None, 256)          0           batch_normalization_11[0][0]
__________________________________________________________________________________________________
dense_5 (Dense)                 (None, 4096)         1052672     tf_op_layer_Relu_11[0][0]
__________________________________________________________________________________________________
tf_op_layer_Add_1 (TensorFlowOp (None, 4096)         0           dense_5[0][0]
__________________________________________________________________________________________________
tf_op_layer_Squeeze_2 (TensorFl (None, 512, 64)      0           tf_op_layer_Relu_6[0][0]
__________________________________________________________________________________________________
tf_op_layer_Reshape_1 (TensorFl (None, 64, 64)       0           tf_op_layer_Add_1[0][0]
__________________________________________________________________________________________________
tf_op_layer_MatMul_1 (TensorFlo (None, 512, 64)      0           tf_op_layer_Squeeze_2[0][0]
                                                                 tf_op_layer_Reshape_1[0][0]
__________________________________________________________________________________________________
tf_op_layer_ExpandDims_2 (Tenso (None, 512, 1, 64)   0           tf_op_layer_MatMul_1[0][0]
__________________________________________________________________________________________________
conv2d_8 (Conv2D)               (None, 512, 1, 64)   4160        tf_op_layer_ExpandDims_2[0][0]
__________________________________________________________________________________________________
batch_normalization_12 (BatchNo (None, 512, 1, 64)   256         conv2d_8[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_12 (TensorFlow (None, 512, 1, 64)   0           batch_normalization_12[0][0]
__________________________________________________________________________________________________
conv2d_9 (Conv2D)               (None, 512, 1, 128)  8320        tf_op_layer_Relu_12[0][0]
__________________________________________________________________________________________________
batch_normalization_13 (BatchNo (None, 512, 1, 128)  512         conv2d_9[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_13 (TensorFlow (None, 512, 1, 128)  0           batch_normalization_13[0][0]
__________________________________________________________________________________________________
conv2d_10 (Conv2D)              (None, 512, 1, 1024) 132096      tf_op_layer_Relu_13[0][0]
__________________________________________________________________________________________________
batch_normalization_14 (BatchNo (None, 512, 1, 1024) 4096        conv2d_10[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_14 (TensorFlow (None, 512, 1, 1024) 0           batch_normalization_14[0][0]
__________________________________________________________________________________________________
max_pooling2d_2 (MaxPooling2D)  (None, 1, 1, 1024)   0           tf_op_layer_Relu_14[0][0]
__________________________________________________________________________________________________
tf_op_layer_Squeeze_3 (TensorFl (None, 1024)         0           max_pooling2d_2[0][0]
__________________________________________________________________________________________________
dense_6 (Dense)                 (None, 512)          524800      tf_op_layer_Squeeze_3[0][0]
__________________________________________________________________________________________________
batch_normalization_15 (BatchNo (None, 512)          2048        dense_6[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_15 (TensorFlow (None, 512)          0           batch_normalization_15[0][0]
__________________________________________________________________________________________________
dropout (Dropout)               (None, 512)          0           tf_op_layer_Relu_15[0][0]
__________________________________________________________________________________________________
dense_7 (Dense)                 (None, 256)          131328      dropout[0][0]
__________________________________________________________________________________________________
batch_normalization_16 (BatchNo (None, 256)          1024        dense_7[0][0]
__________________________________________________________________________________________________
tf_op_layer_Relu_16 (TensorFlow (None, 256)          0           batch_normalization_16[0][0]
__________________________________________________________________________________________________
dropout_1 (Dropout)             (None, 256)          0           tf_op_layer_Relu_16[0][0]
__________________________________________________________________________________________________
dense_8 (Dense)                 (None, 120)          30840       dropout_1[0][0]
==================================================================================================
Total params: 3,512,769
Trainable params: 3,500,609
Non-trainable params: 12,160
__________________________________________________________________________________________________
[90 90 90 90 90 90 90 89 90 90]
[0, 0, 4.738568919164605]
[16/Jan/2023:10:19:35] ENGINE Listening for SIGTERM.
[16/Jan/2023:10:19:35] ENGINE Bus STARTING
CherryPy Checker:
The config entry 'auth.require' is invalid, because the 'auth' config namespace is unknown.
section: [/views]

CherryPy Checker:
'D:\\RS-Reference\\github_code\\SUSTechPOINTS-dev-auto-annotate\\./temp' (root + dir) is not an existing filesystem path.
section: [/temp]
root: 'D:\\RS-Reference\\github_code\\SUSTechPOINTS-dev-auto-annotate'
dir: './temp'

CherryPy Checker:
'D:\\RS-Reference\\github_code\\SUSTechPOINTS-dev-auto-annotate\\./views' (root + dir) is not an existing filesystem path.
section: [/views]
root: 'D:\\RS-Reference\\github_code\\SUSTechPOINTS-dev-auto-annotate'
dir: './views'

CherryPy Checker:
'D:\\RS-Reference\\github_code\\SUSTechPOINTS-dev-auto-annotate\\./assets' (root + dir) is not an existing filesystem path.
section: [/assets]
root: 'D:\\RS-Reference\\github_code\\SUSTechPOINTS-dev-auto-annotate'
dir: './assets'

[16/Jan/2023:10:19:35] ENGINE Set handler for console events.
[16/Jan/2023:10:19:35] ENGINE Started monitor thread 'Autoreloader'.
[16/Jan/2023:10:19:36] ENGINE Serving on http://0.0.0.0:8081
[16/Jan/2023:10:19:36] ENGINE Bus STARTED
127.0.0.1 - - [16/Jan/2023:10:28:04] "GET / HTTP/1.1" 200 51696 "" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36"`
**这是console里的输出，对比了一下正常启动的输出，倒数第6行多了一句“[16/Jan/2023:10:19:35] ENGINE Set handler for console events.”，麻烦你帮忙看一下。**

**Comment by naurril** [comment]
网页的console输出也看一下

**Comment by Mikasal** [comment]
> 网页的console输出也看一下

:8081/static/js/main.js:1 

       Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.

**Comment by Mikasal** [comment]
另外想问一下根据反射强度着色这个功能现在是完善的吗？发现用反射强度着色点云全是黄色，用其他的软件测试相同的点云文件着色是正常的，如果想调试这部分代码请问需要看哪个文件？

**Comment by naurril** [comment]
> 另外想问一下根据反射强度着色这个功能现在是完善的吗？发现用反射强度着色点云全是黄色，用其他的软件测试相同的点云文件着色是正常的，如果想调试这部分代码请问需要看哪个文件？

主要是不同雷达的反射强度取值范围不一样导致的，有的是[0,1], 有的是［0,255].  在lidar.js里面修改．

**Comment by liu-b-s** [comment]
我也遇到同样问题，求教一下，怎么解决的？

**Comment by brillint** [comment]
同问

**Comment by naurril** [comment]
是不是也是windows系统，你用fusion分支，用npm编译一下js的代码试一下

**Comment by jessieWu3** [comment]
> 我也遇到同样问题，求教一下，怎么解决的？
https://github.com/naurril/SUSTechPOINTS/issues/33#issuecomment-957046463
这里，我用这个方法解决了，修改注册表HKEY_CLASSES_ROOT  .js\Content Type 中的'text/plain' 为 'text/javascript'.然后重启就好了
